src/zkevm_specs/evm/execution/callop.py

```diff
@@ -37,7 +45,7 @@ def call_staticcall(instruction: Instruction):
     # Lookup values from stack
     gas_rlc = instruction.stack_pop()
     callee_address_rlc = instruction.stack_pop()
-    # The third argument `value` of opcode CALL is not present for opcode STATICCALL.
+    # The third argument `value` of opcode CALL is not present for both opcode DELEGATECALL and STATICCALL.
     value = instruction.stack_pop() if is_call == 1 else RLC(0)
```

Comment:
Again, not your change. But should this be:
```suggestion
    value = instruction.stack_pop() if is_call == FQ.one() else RLC(0)
```